src/renderer/components/FtSearchFilters/FtSearchFilters.css

```
@@ -18,6 +18,45 @@
   justify-content: center;
 }
 
+.titleContainer {
+  display: flex;
+  align-items: baseline;
+  justify-content: center;
+  gap: 0.5rem;
+  block-size: inherit;
+}
+
+.clearFilterButton {
+  border-radius: 50%;
+  border-style: none;
+  background-color: transparent;
+  color: var(--primary-text-color);
+  cursor: pointer;
+  font-size: 20px;
+  line-height: 1em;
+  padding: 10px;
+  transition: background 0.2s ease-out;
+}
+
+.clearFilterButton,
+:deep(.ftIconButton:not(.arrowDisabled) .iconButton) {
+  &:hover {
+    background-color: var(--side-nav-hover-color);
+    color: var(--side-nav-hover-text-color);
+    transition: background 0.2s ease-in;
+  }
+
+  &:active {
+    background-color: var(--tertiary-text-color);
+    color: var(--side-nav-active-text-color);
+    transition: background 0.2s ease-in;
+  }
+}
+
+.clearFilterIcon {
+  vertical-align: -0.25em;
```

FontAwesome already applies vertical align; this makes the button square.
```suggestion
  inline-size: 1em;
```